homeassistant/components/amcrest/camera.py

```diff
@@ -92,12 +106,129 @@ def __init__(self, hass, amcrest):
         finally:
             await stream.close()
 
+    # Entity property overrides
+
     @property
     def name(self):
         """Return the name of this camera."""
         return self._name
 
+    @property
+    def device_state_attributes(self):
+        """Return the Amcrest-spectific camera state attributes."""
+        return self._static_attrs
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_ON_OFF
```

Comment:
I think you are missing `SUPPORT_STREAM`

Reply:
I'm not adding stream support. That was already done by someone else. But if this flag should be added, I can certainly do that.

Reply:
Understand, but it's a flag that's missing.

Reply:
I just saw that someone else has made that change since I created my branch. I'll merge. Thx!